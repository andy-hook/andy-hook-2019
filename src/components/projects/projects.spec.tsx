import React from "react"

import Projects from "./projects"
import { renderWithTheme } from "@test-utils"

jest.mock("@hooks/page-transition")

jest.mock(
  "@components/shared/projects-grid/projects-grid.container",
  () => "ProjectsGridContainer"
)

describe("<Projects />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <Projects />)
    expect(tree).toMatchSnapshot()
  })
})
