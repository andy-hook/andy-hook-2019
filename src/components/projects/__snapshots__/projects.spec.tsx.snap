// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Projects /> renders correctly 1`] = `
<div
  className="sc-gzVnrw lpvafE"
>
  <div
    className="sc-htpNat cIZlbx"
  >
    <div
      className="sc-ifAKCX gPgsBp"
    >
      <div
        className="sc-htoDjs jKuxOo"
      >
        <h1
          className="sc-bdVaJa dcQDvz"
        >
          Projects
        </h1>
        <h2
          className="sc-bwzfXH bdOJmA"
        >
          A selection of my favouriteÂ work
        </h2>
      </div>
      <ProjectsGridContainer />
    </div>
  </div>
  <footer
    className="sc-bZQynM gUYwJU"
  >
    <div
      className="sc-htpNat cIZlbx"
    >
      <div
        className="sc-ifAKCX gPgsBp"
      >
        I am a footer
      </div>
    </div>
  </footer>
</div>
`;
